src/components/incident/AddTicketUrlSignOffAction.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+
+import SignOffAction, { SignOffActionPropsType } from "./SignOffAction";
+
+type AddTicketUrlPropsType = {
+  onAddTicketUrl: (url: string) => void;
+  signOffActionProps?: Partial<SignOffActionPropsType>;
+};
+
+const AddTicketUrl: React.FC<AddTicketUrlPropsType> = ({
+                                                       onAddTicketUrl,
+                                                       signOffActionProps = {},
+                                                     }: AddTicketUrlPropsType) => {
+
+  const handleSubmit = (url: string) => {
+    onAddTicketUrl(url);
+  };
+
+  return (
+    <SignOffAction
+      dialogTitle="Add ticket URL"
+      dialogContentText="Write an URL of an existing ticket"
+      dialogSubmitText="Submit"
+      dialogCancelText="Cancel"
+      dialogButtonText="Add ticket"
+      title="Add ticket url"
+      question="Are you sure you want to add ticket url?"
```

Comment:
Would this be better as "Are you sure you want to add a ticket url?" I'm not sure, I'm so used to getting rid of the/a in commit messages :)